FAUCETS.md

```diff
@@ -0,0 +1,33 @@
+# Testnet Token Faucets
+
+Rococo:
+
+- https://github.com/paritytech/cumulus/blob/master/parachains/runtimes/contracts/contracts-rococo/README.md#rococo-deployment
+
+- https://app.element.io/#/room/#rococo-faucet:matrix.org
+
+Aleph Zero Testnet:
+
+- https://faucet.test.azero.dev/
+
+- https://docs.alephzero.org/aleph-zero/build/setting-up-testnet-account
+
+Shibuya:
+
+- https://docs.astar.network/docs/build/environment/faucet/
+
+- https://faucet.triangleplatform.com/astar/shibuya
+
+- https://www.as-faucet.xyz/en
+
+Pendulum Testnet:
```

Comment:
let's remove Pendulum for now, couldn't find a faucet link and according to [awesome-ink](https://github.com/paritytech/awesome-ink) there is none